tests/testthat/test-data_read.R

```diff
@@ -516,3 +516,25 @@ test_that("data_read - RDS file, from URL", {
     "Reading data"
   )
 })
+
+
+test_that("data_read - nanoparquet", {
+  skip_if_not_installed("withr")
+  skip_if_not_installed("nanoparquet")
+
+  withr::with_tempfile("temp_file", fileext = ".rda", code = {
```

Comment:
I guess it's a typo: 
```suggestion
  withr::with_tempfile("temp_file", fileext = ".parquet", code = {
```

Reply:
Yeah, copy/paste issue...